obj/ordersList.js

```diff
@@ -0,0 +1,24 @@
+const functions = {
+  navigate: function () {
+    this.click('@ordersHistory');
+  },
+  waitForPage: function () {
+    this.waitForElementVisible('@headline');
+  },
+  searchOrder: function (serachTerm) {
+    this.setValue('@searchField', searchTerm);
+  },
+  assertItemFound: function (searchTerm) {
+    this.waitForElementVisible('firstListItem');
+    this.assert.containsText('@firstListItem', searchTerm);
+  }
+}
+module.exports = {
+  commands: [functions],
+  elements: {
+    ordersHistory: 'aside .list--dense:nth-child(3) div:nth-of-type(3) span',
```

Comment:
In this case it would be better to use xpath with text instead. This one is kind of unstable.